spanner/sessionclient.go

```diff
@@ -272,5 +276,31 @@ func (sc *sessionClient) nextClient() (*vkit.Client, error) {
 	if err != nil {
 		return nil, err
 	}
+	if sc.callOptions != nil {
+		client.CallOptions = mergeCallOptions(client.CallOptions, sc.callOptions)
+	}
 	return client, nil
 }
+
+// mergeCallOptions merges two CallOptions into one and the first argument has
+// a lower order of precedence than the second one.
+func mergeCallOptions(low *vkit.CallOptions, high *vkit.CallOptions) *vkit.CallOptions {
```

Comment:
nit: I find the `low` and `high` variable names here a little confusing. It kind of indicates that it will give precedence to the highest or lowest value, while it is just `a` and `b`, where `b` takes precedence. So could we rename it to either `a` and `b` or `first` and `second` (or something like that)?

Reply:
That's a good catch. Originally, I used `src` and `dest`, but then I feel it is not quite right and changed to `low` and `high`. I will go for `a` and `b`.